Prace_domowe/Praca_domowa2/Grupa4/Patryk_Tomaszewski/PD2.ipynb

```diff
@@ -0,0 +1,1574 @@
+{
```

Comment:
<p>Jak to wartości std fluktuują? Czyli puściłeś kilka razy po 10 i std ma rozpiętość&nbsp;od 1300 do 3900, tak?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/74/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>